package.json

```diff
@@ -66,7 +66,7 @@
   "scripts": {
     "serve": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --coverage",
```

Comment:
Revert this change. It works for our GitHub actions.

Reply:
done! i removed the coverage and pushed